tests/cursorid/cursorid-002.phpt

```diff
@@ -37,7 +37,8 @@ printf("Killed expected cursor: %s\n", (string) $cursorId === (string) $result->
 ?>
 ===DONE===
 <?php exit(0); ?>
---EXPECT--
+--EXPECTF--
+Deprecated: MongoDB\Driver\Cursor::getId(): The method "MongoDB\Driver\Cursor::getId" will no longer return a "MongoDB\Driver\CursorId" instance in the future. Pass "true" as argument to change to the new behavior and receive a "MongoDB\BSON\Int64" instance instead. in %s
 Killed 1 cursor(s)
 Killed expected cursor: yes
```

Comment:
I imagine `$result->cursorsKilled[0]` will always come back as a PHP integer unless we're on a 32-bit platform. The existing comparison using string casts and `===` can probably stay, since it'd still be necessary if `$cursorId` was an Int64 instead of a CursorId object.

Reply:
@alcaeus: I think you may want to revert this if the purpose of this test was to assert BSON serialization of _CursorId_ instances. Related to my comment in https://github.com/mongodb/mongo-php-driver/pull/1616#discussion_r1743951768.

Reply:
Agree. I've reverted the change.